### PR TITLE
Update installation.rst

### PR DESCRIPTION
**Pull request by drewmassey**
Clarified rewrite setup for nginx

### CODE REVIEW

File: en/installation.rst [hunk 1]
```diff
@@ -399,6 +399,37 @@ you will need PHP running as a FastCGI instance::
             fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
         }
     }
+    
+One some servers (Like Ubuntu 12.04) the above configuration won't work out of the box,
```
Comment (ADmad) [inline en/installation.rst]: It would be better to reference the latest LTS which is 14.04.  

Also please limit text line lengths to 80 columns.

Comment (antograssiot) [inline en/installation.rst]: One -> On some servers...

Comment (drewmassey) [inline en/installation.rst]: Oops, actually this was on Ubuntu 14.10. Sorry about that.

**Comment by markstory** [comment]
Cleaned up and merged in.

**Pull request merged**